pkg/gameservers/portallocator_test.go

```diff
@@ -123,29 +125,30 @@ func TestPortAllocatorAllocate(t *testing.T) {
 		assert.True(t, cache.WaitForCacheSync(stop, pa.nodeSynced))
 
 		err := pa.syncAll()
-		assert.Nil(t, err)
+		require.NoError(t, err)
 
 		// two nodes
 		for x := 0; x < 2; x++ {
-			// ports between 10 and 20
-			for i := 10; i <= 20; i++ {
+			for i := 0; i < 11; i++ {
 				var p int32
 				gs := pa.Allocate(fixture.DeepCopy())
+				assert.NotNil(t, gs)
 				assert.True(t, 10 <= gs.Spec.Ports[0].HostPort && gs.Spec.Ports[0].HostPort <= 20, "%v is not between 10 and 20", p)
-				assert.Nil(t, err)
 			}
 		}
 
 		assert.Len(t, pa.portAllocations, 2)
 		gs := pa.Allocate(fixture.DeepCopy())
+		assert.NotNil(t, gs)
 		assert.NotEmpty(t, gs.Spec.Ports[0].HostPort)
 		assert.Len(t, pa.portAllocations, 3)
 	})
 
 	t.Run("ports are all allocated with multiple ports per GameServers", func(t *testing.T) {
 		m := agtesting.NewMocks()
-		maxPort := int32(19) // make sure we have an even number
-		pa := NewPortAllocator(10, maxPort, m.KubeInformerFactory, m.AgonesInformerFactory)
+		minPort := int32(10)
+		maxPort := int32(20) // make sure we have an even number
```

Comment:
```suggestion
		maxPort := int32(19) // make sure we have an even number
```
Now we have odd number of ports. Not sure what we should change this comment or set back the number and consequently other comments with `20`.

Reply:
I thought this comment is related to maxPort that is why I’ve changed it.

Reply:
Outside of this comment, I think this PR is good to merge?

Reply:
Yes.
Are you aware of a context of this comment?
If we all are unsure about it I think we can remove one and merge.

Reply:
Looks like I wrote it.... and I can't remember anymore :confused:

Reply:
Updated